### PR TITLE
#1 Implement minimal http-server

### PR DESCRIPTION
**Pull request by zimash**

### CODE REVIEW

**Comment by zimash** [comment]
@torwald-sergesson need a review

File: main.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	fmt.Fprint(w, jsonPayload{})
+}
+
+func main() {
+	flag.IntVar(&httpPort, "port", 8080, "Port to serve")
+	flag.Parse()
+
+	http.HandleFunc("/healthz", healthCheck)
+	log.Printf("Server starting on %d port", httpPort)
+	http.ListenAndServe(fmt.Sprintf(":%d", httpPort), nil)
```
Comment (torwald-sergesson) [inline main.go]: Handle returned error (at least with log.Fatal)

File: main.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+var (
+	httpPort int
+)
+
+type jsonPayload struct{}
+
+func healthCheck(w http.ResponseWriter, r *http.Request) {
+	log.Printf("%s from %s\n", r.RequestURI, r.RemoteAddr)
+	w.Header().Add("Content-Type", "application/json")
+	fmt.Fprint(w, jsonPayload{})
```
Comment (torwald-sergesson) [inline main.go]: Handle returned error to log.Printf

**Review by torwald-sergesson — COMMENTED** [review]
Some comments to fix

File: main.go (hunk 2 above)
Comment (zimash) [reply main.go]: Fixed

File: main.go (hunk 1 above)
Comment (zimash) [reply main.go]: Fixed

**Review by torwald-sergesson — APPROVED** [review]

**Pull request merged**